app/javascript/listings/dashboard/listingRow.jsx

```diff
@@ -1,5 +1,6 @@
 import PropTypes from 'prop-types';
-import { h } from 'preact';
+import { h, Fragment } from 'preact';
```

Comment:
Nice! I'm reacquainting myself with React (and learning more about Preact!) and I hadn't heard of `Fragment` components before! Pretty neat; thanks for teaching me about them by using them in your PR 😸